### PR TITLE
feat: bind the Node to insecure unix socket

### PR DESCRIPTION
**Pull request by sshaman1101**
Added:
- Insecure gRPC client and server constructor that can be binded to unix socket.

Changed:
- Node binds to unix socket, CLI connect via unix socket.

Removed:
- CLI do not require transport creds for gRPC.

### CODE REVIEW

File: util/xgrpc/client.go [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
 	"github.com/grpc-ecosystem/go-grpc-middleware/tracing/opentracing"
 	"github.com/opentracing/basictracer-go"
+	"github.com/opentracing/opentracing-go"
 	"github.com/sonm-io/core/insonmnia/auth"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 )
 
+func getTracer() opentracing.Tracer {
```
Comment (3Hren) [inline util/xgrpc/client.go]: newTracer

File: cmd/cli/commands/common.go [hunk 2]
```diff
@@ -46,14 +42,13 @@ var (
...
 	sessionKey *ecdsa.PrivateKey = nil
-	creds      credentials.TransportCredentials
 
 	// errors
 	errCannotParsePropsFile = errors.New("cannot parse props file")
 )
 
 func init() {
-	rootCmd.PersistentFlags().StringVar(&nodeAddressFlag, "node", "127.0.0.1:15030", "node addr")
+	rootCmd.PersistentFlags().StringVar(&nodeAddressFlag, "node", "/tmp/sonm_node.sock", "node socket")
```
Comment (3Hren) [inline cmd/cli/commands/common.go]: /var/run/...sock

File: node.yaml [hunk 3]
```diff
@@ -1,8 +1,7 @@
 # Local Node settings
 node:
-  # address to bind Node's gRPC endpoint
-  # required
-  listen_addr: "127.0.0.1:15030"
+  # socket path to bind Node's gRPC endpoint, required
+  socket_path: "/tmp/sonm_node.sock"
```
Comment (3Hren) [inline node.yaml]: Also make default value.

File: insonmnia/node/server.go [hunk 4]
```diff
@@ -178,3 +173,11 @@ func (n *Node) Serve() error {
 
 	return n.srv.Serve(n.lis)
 }
+
+func (n *Node) Close() {
+	if n.lis != nil {
+		if err := n.lis.Close(); err != nil {
```
Comment (antmat) [inline insonmnia/node/server.go]: does this unlink socket?

Comment (sshaman1101) [reply insonmnia/node/server.go]: Yes.

**Pull request merged**